### PR TITLE
IN-415

### PR DESCRIPTION
**Pull request by tushar-balwani**
# Description

Updated create remediation scan API to support enabling of selected plugins.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- Added below test case to add list of plugin ids to enable in remediation scan
- [x] `test_remedyscan_enable_selected_plugins` 

**Test Configuration**:
* Python Version(s) Tested: 3.8.6
* Tenable.sc version (if necessary):

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tenable/io/remediation_scans.py [hunk 1]
```diff
@@ -3,15 +3,15 @@
 =================
 
 The following methods allow for interaction into the Tenable.io
-:devportal:`scans <Create Remediation Scans>` API endpoints.
-:devportal:`scans <List Remediation Scans>` API endpoints.
```
[the remark this answers is not in the document]
Comment (tushar-balwani) [reply tenable/io/remediation_scans.py:7]: These APIs are under scans as per API docs. 
I have added Scans as parent link in above description and API links in respective methods

[the remark this answers is not in the document]
Comment (mvalliammal) [reply tenable/io/remediation_scans.py:7]: It should not be scans end point in the devportal
It needs to be remediationscan

[the remark this answers is not in the document]
Comment (tushar-balwani) [reply tenable/io/remediation_scans.py:7]: reverted devportal links to existing links in remediation scans docs

**Review by mvalliammal — APPROVED** [review]
I approved

**Pull request merged**